doc/whats_new.rst

```diff
@@ -18,6 +18,9 @@ Changelog
 - Changed ``conn_seed`` default to ``None`` (from ``3``) in :func:`~hnn_core.network.add_connection`, 
   by `Mattan Pelah`_ in :gh:`492`.
 
+- Add interface to modify attributes of sections in
+ :func:`~hnn_core.viz.plot_morphology`, by `Nick Tolley`_ in :gh:`481`
+
```

Comment:
I think this should mention `Cell.modify_section` right?

Reply:
oops! typo